notebooks/ch12/LR_Case_Study.md

```diff
@@ -0,0 +1,792 @@
+      <td>30</td>
+      <td>41.2</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>1</td>
+      <td>43.0</td>
+      <td>33.8</td>
+      <td>4</td>
+      <td>41</td>
+      <td>31.7</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>1</td>
+      <td>48.0</td>
+      <td>40.6</td>
+      <td>3</td>
+      <td>38</td>
+      <td>39.4</td>
+    </tr>
+    <tr>
+      <th>...</th>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+    </tr>
+    <tr>
+      <th>43</th>
+      <td>1</td>
+      <td>44.0</td>
+      <td>35.3</td>
+      <td>3</td>
+      <td>32</td>
+      <td>51.6</td>
+    </tr>
+    <tr>
+      <th>44</th>
+      <td>1</td>
+      <td>63.0</td>
+      <td>39.3</td>
+      <td>8</td>
+      <td>40</td>
+      <td>55.7</td>
+    </tr>
+    <tr>
+      <th>45</th>
+      <td>1</td>
+      <td>58.0</td>
+      <td>43.8</td>
+      <td>13</td>
+      <td>57</td>
+      <td>55.5</td>
+    </tr>
+  </tbody>
+</table>
+<p>46 rows × 6 columns</p>
+</div>
+
+
+
+Let's deal with that pesky null value. We can use the `dropna` function for this purpose.
+
+
+```python
+drinking_data = drinking_data.dropna()
+drinking_data
+```
+
+
+
+
+<div>
+<style>
+    .dataframe thead tr:only-child th {
+        text-align: right;
+    }
+
+    .dataframe thead th {
+        text-align: left;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>Bias</th>
+      <th>Urban population</th>
+      <th>Late births</th>
+      <th>Wine</th>
+      <th>Liquor</th>
+      <th>Cirrhosis</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>1</td>
+      <td>44.0</td>
+      <td>33.2</td>
+      <td>5</td>
+      <td>30</td>
+      <td>41.2</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>1</td>
+      <td>43.0</td>
+      <td>33.8</td>
+      <td>4</td>
+      <td>41</td>
+      <td>31.7</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>1</td>
+      <td>48.0</td>
+      <td>40.6</td>
+      <td>3</td>
+      <td>38</td>
+      <td>39.4</td>
+    </tr>
+    <tr>
+      <th>...</th>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+    </tr>
+    <tr>
+      <th>43</th>
+      <td>1</td>
+      <td>44.0</td>
+      <td>35.3</td>
+      <td>3</td>
+      <td>32</td>
+      <td>51.6</td>
+    </tr>
+    <tr>
+      <th>44</th>
+      <td>1</td>
+      <td>63.0</td>
+      <td>39.3</td>
+      <td>8</td>
+      <td>40</td>
+      <td>55.7</td>
+    </tr>
+    <tr>
+      <th>45</th>
+      <td>1</td>
+      <td>58.0</td>
+      <td>43.8</td>
+      <td>13</td>
+      <td>57</td>
+      <td>55.5</td>
+    </tr>
+  </tbody>
+</table>
+<p>45 rows × 6 columns</p>
+</div>
+
+
+
+Looks like it worked! We now have 45 rows instead of 46 rows. 
```

Comment:
maybe we can just look at the 35th element again

Reply:
hmm i think i like it the way it is